mayavi/components/implicit_plane.py

```diff
@@ -149,15 +149,18 @@ def update_plane(self):
     ######################################################################
     def _get_normal(self):
         return self.widget.normal
+
     def _set_normal(self, value):
         w = self.widget
         old = w.normal
         w.normal = value
         self.trait_property_changed('normal', old, value)
         self.update_plane()
+        self.pipeline_changed = True
```

Comment:
Now that you have some tests, it may be good to see if there is a less performance intense approach to fix this.  A pipeline change is costly as it flushes everything downstream, i.e. all connections are re-made.  A data_changed is cheaper but may not do the trick, perhaps an explicit update call somewhere may help, I am not sure but `pipeline_changed` is costly.